fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java

```diff
@@ -1318,6 +1342,18 @@ public List<String> visitIdentifierSeq(IdentifierSeqContext ctx) {
             .collect(ImmutableList.toImmutableList());
     }
 
+    @Override
+    public Pair<List<String>, Expression> visitUpdateAssignment(UpdateAssignmentContext ctx) {
+        return Pair.of(visitMultipartIdentifier(ctx.multipartIdentifier()), getExpression(ctx.expression()));
+    }
```

Comment:
why not gernate `Equals(unboundSlot, Expression)`

Reply:
EqualTo is also ok.